### DISCUSSION TITLE
Any means to charge customer for usage time?

### DISCUSSION THREAD

**Discussion by neilyoung** [bluenviron/mediamtx#1357] [Questions]
As subject says. I know there is no charging interface, but what other ways would be available to charge customers per usage?

**Comment by neilyoung** [comment]
Well, log parsing...

**Comment by aler9** [comment]
You can use the API to fetch the `created` field of each user in order to find out how long the user has been connected.

**Reply by neilyoung** [reply]
Thanks, but as a matter of fact: Parsing your log works perfectly. I hope this will not see too many changes in the future.

[a comment by an automated account is not shown]
